packages/opentelemetry-core/src/common/time.ts

```diff
@@ -117,6 +117,7 @@ export function hrTimeDuration(
  * Convert hrTime to timestamp, for example "2019-05-14T17:00:00.000123456Z"
  * @param hrTime
  */
+// eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
I think it would be better to use a different name than to disable no-shadow

Reply:
I thought about it, but having a parameter called `hrTime` tell a lot to Javascript (not Typescript) library users. We could also call it `hrtime` but I still think just disabling is better than sacrificing the name.

Reply:
ok